src/components/Movie/MovieBanner.js

```diff
@@ -0,0 +1,37 @@
+import React, {Component} from 'react';
+
+import PropTypeShapes from '../Common/PropTypeShapes';
+
+const MovieBanner = (props) => {
+    const { poster_path, title, vote_average, genres, release_date, runtime, overview } = props.movie;

```

Comment:
When you have a long list of parameters in the destructuring, is better use multiple lines to describe that. Also, for property names, always prefer `camelCase`.

```javascript
const {
    poster_path,
    title,
    ...otherParams
} = props.movie;
```

Reply:
Done